### PR TITLE
Fix/obsolete hex conversion & replace socket conversion consistently

### PR DESCRIPTION
**Pull request by AddressXception**
removes the references for `FromHex` in favor of the builtin method. Move the socket protocol string replacement for the bridge url into the `GetBridgeUrl` method so that it is consistently applied from both call sites.

### CODE REVIEW

File: WalletConnectSharp.Core/Network/DefaultBridge.cs [hunk 1]
```diff
@@ -59,7 +59,14 @@ public static string ChooseRandomBridge(string[] possibleBridges = null)
     public static string GetBridgeUrl(string url)
     {
         if (ExtractRootDomain(url) == Domain)
-            return ChooseRandomBridge();
+        {
+            var chosen = ChooseRandomBridge();
+            if (url.StartsWith("wss"))
+                chosen = chosen.Replace("https", "wss");
+            else if (url.StartsWith("ws"))
+                chosen = chosen.Replace("http", "ws");
```
Comment (john-s-morgan) [inline WalletConnectSharp.Core/Network/DefaultBridge.cs:67]: Just to get rid of the magic strings...

```suggestion
            if (url.StartsWith(Uri.UriSchemeWss))
                chosen = chosen.Replace(Uri.UriSchemeHttps, Uri.UriSchemeWss);
            else if (url.StartsWith(Uri.UriSchemeWs))
                chosen = chosen.Replace(Uri.UriSchemeHttp, Uri.UriSchemeWs);
```

Comment (ecp4224) [reply WalletConnectSharp.Core/Network/DefaultBridge.cs]: I like the idea of using the `Uri.UriSchemeWss` variables, however I think the variables may need to be imported or maybe their called something else. The build broke :(

File: WalletConnectSharp.Core/Network/DefaultBridge.cs [hunk 2]
```diff
@@ -59,7 +59,14 @@ public static string ChooseRandomBridge(string[] possibleBridges = null)
     public static string GetBridgeUrl(string url)
     {
         if (ExtractRootDomain(url) == Domain)
-            return ChooseRandomBridge();
+        {
+            var chosen = ChooseRandomBridge();
+            if (url.StartsWith(Uri.UriSchemeWss))
```
Comment (ecp4224) [inline WalletConnectSharp.Core/Network/DefaultBridge.cs:64]: Please change magic strings to variables

**Review by ecp4224 — CHANGES_REQUESTED** [review]

File: WalletConnectSharp.Core/Network/DefaultBridge.cs (hunk 1 above)
Comment (john-s-morgan) [reply WalletConnectSharp.Core/Network/DefaultBridge.cs]: Yeah, it looks like [UriSchemeWSS](https://docs.microsoft.com/en-us/dotnet/api/system.uri.urischemewss) is .NET 6 forward, so netstandard2.1 isn't getting it just yet 😢 

Will need to define a static readonly class or a file-level constant to remove strings here.

**Review by ecp4224 — APPROVED** [review]

**Pull request merged**